server/src/resources/js/checkout.js

```diff
@@ -72,5 +74,65 @@ class Checkout {
                 console.log(error);
             });
     }
+
+    getBasketTotal() {
+        var userToken = localStorage.getItem('userToken');
+
+        let query = `{
+            me {
+                cart {
+                    cart_total
+                }
+            }
+        }`;
+
+        return this.graphqlService
+            .sendQuery(query, userToken)
+            .then(re => {
+                let currency = localStorage.getItem('currency');
+
+                // A naive solution for now, but if now currency was specified by user then I'll just assume the currency to be
+                // USD. This is definitely TODO: improve
+                if (!currency) {
```

Comment:
@AlexeiaW  as per other comment, we need to just store this in config and pull it out during the graphql shop config query as "currency_default" or something along those lines.